invenio_users_resources/services/groups/config.py

```diff
@@ -25,8 +29,33 @@
 class GroupSearchOptions(SearchOptions):
     """Search options."""
 
-    # TODO search params
-    params_interpreters_cls = SearchOptions.params_interpreters_cls
+    pagination_options = {
+        "default_results_per_page": 10,
+        "default_max_results": 10,
+    }
+
+    query_parser_cls = QueryParser.factory(
+        fields=["name", "title", "description"]
+    )
+
+    sort_default = 'bestmatch'
+    sort_default_no_query = 'name'
+    sort_options = {
+        "bestmatch": dict(
+            title=_('Best match'),
+            fields=['_score'],  # ES defaults to desc on `_score` field
+        ),
+        "name": dict(  # TODO: add asc/desc
```

Comment:
What does `asc/desc` mean exactly?

Reply:
it's sorting by `name` but we might also want ascending and descending (alphabetical and inverse alphabetical order, default is ascending)